mappings/net/minecraft/entity/ai/TargetPredicate.mapping

```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/class_4051 net/minecraft/entity/ai/TargetPredicate
 	FIELD field_18096 includeHidden Z
 	FIELD field_18097 ignoreEntityTargetRules Z
 	FIELD field_18098 useDistanceScalingFactor Z
+	FIELD field_28405 predicate Ljava/util/function/Predicate;
```

Comment:
Is this called `predicate` in json form?

Reply:
What json form?

Reply:
oh it's ai, nevermind